Makefile

```diff
@@ -0,0 +1,60 @@
+dev:
+	$(eval DEPLOY_ENV=dev)
+	$(eval AZURE_SUBSCRIPTION=s165-teachingqualificationsservice-development)
+
+test:
+	$(eval DEPLOY_ENV=test)
+	$(eval AZURE_SUBSCRIPTION=s165-teachingqualificationsservice-test)
+
+preprod:
+	$(eval DEPLOY_ENV=preprod)
+	$(eval AZURE_SUBSCRIPTION=s165-teachingqualificationsservice-test)
+
+production:
+	$(eval DEPLOY_ENV=production)
```

Comment:
This needs to be prod. Environment name is defined as prod. Or change environment name on Repo and change all the references of prod to production.